core/services/scan.go

```diff
@@ -279,9 +279,6 @@ func generateScanID(workload domain.ScanCommand) string {
 	if workload.InstanceID != "" {
 		return workload.InstanceID
 	}
-	if workload.ImageTag != "" && workload.ImageHash != "" {
```

Comment:
either this, or we hash the string... @dwertent

Reply:
I think we should hash the string

Reply:
@matthyx